### DISCUSSION TITLE
index.html 消失

### DISCUSSION THREAD

**Discussion by WForst-Breeze** [YunYouJun/hexo-theme-starry#379] [Q&A 疑难杂症] [answered]
本人零基础 TwT，在添加了一些诸如 tags 和 about 等基础的页面后执行 `hexo generate`，但当我进行测试的时候发现，首页消失报错404了（呜哇
后来翻了翻文件夹，发现在 public 文件夹中的 index.html 消失了，并且我的网站默认语言还变成日语了……然而我在 _config.yml 中设置的是 zh_CN 没错啊qwq
麻烦作者大大解答~辛苦啦！

**Comment by YunYouJun** [comment]
请提供可复现仓库。

---

[valaxy](https://github.com/YunYouJun/valaxy) 与 [valaxy-theme-yun](https://github.com/YunYouJun/valaxy/tree/main/packages/valaxy-theme-yun) 已基本可用，推荐可尝试直接使用，初始化模版与构建更加便捷，热更新也更为流畅。

**Reply by WForst-Breeze** [reply]
https://github.com/WForst-Breeze/Files/tree/main/WebHexo

**Comment by FallonWaagvkp** [comment]
It's possible that there may be a setting overriding your zh_CN setting in _config.yml. You may want to check other configuration files or language settings to see if there's anything that could be causing the issue.
To troubleshoot further, you may want to try regenerating your site with Hexo.

**Reply by WForst-Breeze** [reply]
I did not find the problem you mentioned, and after I regenerated the site, I still could not generate index.html😥but thanks

**Comment by YunYouJun** [comment, marked as answer]
找到原因了，你没有新建文章，你需要在 `source/_posts` 下新建你的第一篇文章，譬如 `source/_posts/xxx.md`。

**Reply by WForst-Breeze** [reply]
！太感谢啦www
